pageserver/src/config.rs

```diff
@@ -138,9 +138,16 @@ pub struct PageServerConf {
 
     pub pg_distrib_dir: PathBuf,
 
-    pub auth_type: AuthType,
-
+    // Authentication
+
+    /// authentication method for the HTTP mgmt API
+    pub mgmt_auth_type: AuthType,
```

Comment:
With more values related to auth configuration I'm starting to think that it can be a good idea to move them into separate AuthConfig. WDYT?

Reply:
I hope we won't grow too many of them..

Reply:
Yeah, me too

Separation can be attempted separately if you'd like to postpone it